File: app/src/main/java/com/wolcano/musicplayer/music/utils/NavigationViewUtils.java
package com.wolcano.musicplayer.music.utils;

import android.content.res.ColorStateList;

import androidx.annotation.ColorInt;
import androidx.annotation.NonNull;

import com.google.android.material.bottomnavigation.BottomNavigationView;

public class NavigationViewUtils {
    public static void setItemIconColors(@NonNull BottomNavigationView navigationView, @ColorInt int normalColor, @ColorInt int selectedColor) {
        ColorStateList iconSl = new ColorStateList(new int[][]{{-16842912}, {16842912}}, new int[]{normalColor, selectedColor});
        navigationView.setItemIconTintList(iconSl);
    }

    public static void setItemTextColors(@NonNull BottomNavigationView navigationView, @ColorInt int normalColor, @ColorInt int selectedColor) {
        ColorStateList textSl = new ColorStateList(new int[][]{{-16842912}, {16842912}}, new int[]{normalColor, selectedColor});
        navigationView.setItemTextColor(textSl);
    }

    private NavigationViewUtils() {
    }
}

File: app/src/main/java/com/wolcano/musicplayer/music/utils/NavigationViewUtils.kt
package com.wolcano.musicplayer.music.utils

import android.content.res.ColorStateList
import androidx.annotation.ColorInt
import com.google.android.material.bottomnavigation.BottomNavigationView

object NavigationViewUtils {

    fun setItemIconColors(
        navigationView: BottomNavigationView,
        @ColorInt normalColor: Int,
        @ColorInt selectedColor: Int
    ) {
        val iconSl = ColorStateList(
            arrayOf(intArrayOf(-16842912), intArrayOf(16842912)),
            intArrayOf(normalColor, selectedColor)
        )
        navigationView.itemIconTintList = iconSl
    }

    fun setItemTextColors(
        navigationView: BottomNavigationView,
        @ColorInt normalColor: Int,
        @ColorInt selectedColor: Int
    ) {
        val textSl = ColorStateList(
            arrayOf(intArrayOf(-16842912), intArrayOf(16842912)),
            intArrayOf(normalColor, selectedColor)
        )
        navigationView.itemTextColor = textSl
    }

}